classlib/src/main/java/org/teavm/classlib/java/util/TLinkedHashMap.java

```diff
@@ -545,4 +373,34 @@ public void clear() {
         head = null;
         tail = null;
     }
+
+    @Override
+    public void replaceAll(BiFunction<? super K, ? super V, ? extends V> function) {
+        if (elementCount > 0) {
+            int prevModCount = modCount;
+            TLinkedHashMap.LinkedHashMapEntry<K, V> entry = head;
```

Comment:
I don't think `TLInkedHashMap.` prefix is necessary here. Anyway, I do use `var' in such sutiations